tests/phpunit/tests/mail/test-phpmailer-translations.php

```diff
@@ -1,18 +1,64 @@
 <?php
 
 class Test_PHPMailer_Translations extends WP_UnitTestCase {
+
+	private $original_locale;
```

Comment:
Add some comments for the class attributes. You can see some example here,

https://github.com/WordPress/wordpress-develop/blob/32880ec4e910837cd267e1685733867b243e22fa/src/wp-includes/PHPMailer/PHPMailer.php#L86-L106

Address at all places.